R/validate_inputs.R

```diff
@@ -0,0 +1,164 @@
+
+#' send input validation messages to output
+#'
+#' Captures messages from \code{InputValidator} objects and collates them
```

Comment:
```suggestion
#' Captures messages from \code{shinyvalidate::InputValidator} objects and collates them
```

Reply:
I think it would be a good idea to link to `shinyvalidate` functions as they are external to `teal`, what do you think @chlebowa ? Maybe not all of them but at least the description. For this we can use the format `[shinyvalidate::InputValidator()]`

Reply:
and is there a reason to use `\code{}` here? I do see any instance of this usage in `teal`

Reply:
@chlebowa `\code{}` is the old way of assigning code, we now use ``, hence there is no occurence of `\code{}` in `teal`.
Could you please re-open the PR and correct them to conserve the good practice?